lib/commonBindings.js

```diff
@@ -92,10 +92,7 @@ function guessType(attribute, device) {
     }
 
     if (attribute === constants.TIMESTAMP_ATTRIBUTE) {
-        if (iotAgentLib.configModule.isCurrentNgsi()) {
-            return constants.TIMESTAMP_TYPE_NGSI2;
-        }
-        return constants.TIMESTAMP_TYPE;
```

Comment:
If `TIMESTAMP_TYPE` constant is not used in any place, we could remove it from the source file.

Reply:
Fixed in https://github.com/telefonicaid/iotagent-json/pull/533/commits/c904dfbc857e2fbcef584e279998f7d52e838d92